src/components/LayoutEmbed.js

```diff
@@ -177,24 +178,25 @@ export default function Layout({
           <SearchFilter />
         </Box>
       )}
-      <Box
-        onClick={handleFullScreen}
-        sx={{
-          position: 'absolute',
-          zIndex: '9999',
-          top: '0px',
-          right: '0px',
-          margin: '20px',
-          cursor: 'pointer',
-        }}
-      >
-        <SvgIcon
-          component={MinIcon}
-          inheritViewBox
-          sx={{ height: 52, width: 52 }}
-        />
-      </Box>
-
+      {useEmbed || (
```

Comment:
@okdv is `useEmbed` a function, if true, why you just use function name rather than executing it for checking?

Reply:
Why are we checking for an embed on the embed layout?

Reply:
@dadiorchen  ah yup you are correct, it should be treated as a function. just updated that and submitting a PR for it 

@RubenSmn The naming is a little ambiguous so perhaps that should be updated to make things less confusing. But basically useEmbed is checking for the embed=true parameter and doesnt look at the localStorage. This way we can hide the un/embed button when embed=true is a parameter, but still show when embed is only true in localStorage (i.e. a user has clicked the embed/expand map button, so we want to leave the untoggle button)